drivers/usb/device/usb_dc_sam_usbhs.c

```diff
@@ -348,7 +358,7 @@ int usb_dc_attach(void)
 int usb_dc_detach(void)
 {
 	/* Detach the device */
-	USBHS->USBHS_DEVCTRL &= ~USBHS_DEVCTRL_DETACH;
+	USBHS->USBHS_DEVCTRL |= USBHS_DEVCTRL_DETACH;
```

Comment:
This change seems to deserve a separate commit.

Reply:
Done.